src/explore-education-statistics-frontend/src/components/CookieBanner.tsx

```diff
@@ -12,38 +13,106 @@ interface Props {
 
 function CookieBanner({ wide }: Props) {
   const { getCookie, setBannerSeenCookie, setGADisabledCookie } = useCookies();
+  const isBannerSeen = getCookie('bannerSeen') === 'true';
+  const [isVisible, setVisible] = useState(!isBannerSeen);
+  const { isMounted } = useMounted();
 
-  const { isMounted } = useMounted(() => {
-    if (getCookie('disableGA') === 'true') {
-      setGADisabledCookie(true);
-    }
-  });
+  useEffect(() => {
+    setVisible(!isBannerSeen);
+  }, [isBannerSeen]);
 
-  return isMounted && getCookie('bannerSeen') !== 'true' ? (
+  return isMounted && isVisible ? (
     <div className={styles.container}>
       <div
-        className={classNames('govuk-width-container', 'dfe-width-container', {
-          'dfe-width-container--wide': wide,
-        })}
+        className={classNames(
+          'govuk-cookie-banner',
+          'govuk-width-container',
+          'dfe-width-container',
+          {
+            'dfe-width-container--wide': wide,
+          },
+        )}
+        role="region"
+        aria-label="Cookies on Explore Education Statistics"
       >
-        <p>
-          <span>GOV.UK uses cookies to make the site simpler.</span>{' '}
-          <ButtonText
-            onClick={() => {
-              setBannerSeenCookie(true);
-              if (typeof getCookie('disableGA') === 'undefined') {
-                setGADisabledCookie(false);
-              }
-            }}
+        {typeof getCookie('disableGA') === 'undefined' && (
```

Comment:
Could extract this out into a `isGADisabled` variable like the `isBannerSeen`

Reply:
Since it's only used in one place with the ternary being used, I'll not pull it out. Unless you insist :)

Reply:
By removing redundant checks, there is only one check whether the disableGA cookie is undefined now, so I won't pull it out, unless you argue it :)

Reply:
Nope everything looks pretty clean now! :100: